src/api/beers.js

```diff
@@ -1,18 +1,18 @@
 import axios from 'axios';
 
 export const fetchBeers = () => (
-  axios.get('https://api.punkapi.com/v2/beers')
+  axios.get('https://api.punkapi.com/v2/beers') // This should be set as a constant 
     .then(response => {
       const { data } = response;
       const beers = {};
 
-      data.forEach(beer => {
-        beers[beer.id] = beer
+      data.forEach(beer => { // Use Array HOF as mush as possible
```

Comment:
`forEach` is also one of the HOF. But I understand the best one to have used here is `map`